_partials/_timescale-value-prop.md

```diff
@@ -0,0 +1,61 @@
+### Vastly improve query performance
+
+As the amount of data in your database grows, query speed tends to get slower.
+Timescale hypertables can dramatically improve the performance of large tables.
+Hypertables work just like standard PostgreSQL tables, but they are broken down
+into chunks, and automatically partitioned by time. This gives you improved
+insert and query performance, plus access to an entire suite of useful tools.
+Because you work with hypertables in the same way as standard PostgreSQL tables,
+you don't need to re-architect your application, or learn any new languages.
+
+### Automatically refresh materialized views with continuous aggregates
+
+Time-series data grows very quickly, and as the data grows, analyzing it gets
+slower and uses more resources. Timescale solves the slow-down with continuous
+aggregates. Based on PostgreSQL materialized views, continuous aggregates are
+incrementally and continuously updated, to make them lightning fast.
+Materialized views need to be refreshed and recalculated from scratch every time
+they are run, but continuous aggregates are incrementally updated in the
+background, so they do not require a lot of resources to keep up to date.
+Additionally, you can query your continuous aggregates even if the underlying
+data is compressed and archived.
+
+### Achieve 90%+ compression
+
+It is common for databases to provide compression that saves space, but doesn't
+improve query performance for data that spans long time intervals. Timescale
+includes native compression for your hypertables that improves query performance
+and dramatically reduces data volume. Timescale compression uses native columnar
+compression and best-in-class algorithms that are automatically chosen based on
+your data, which saves space and improves performance. Timescale subscribers
+usually see compression ratios greater than 90% and, because you only pay for
+the storage space that you actually use, that means that you save money from the
+moment you start.
+
+### Lower costs with data tiering to S3
+
+When you are working with time-series and event data, storage costs can easily
+spiral out of control. With Timescale, you never have to worry about hidden
```

Comment:
Maybe talk more about "data-intensive applications that collect large amounts of time-series and events data".